src/commcare_cloud/ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -39,12 +39,12 @@ discovery.zen.fd.ping_interval: 10s
 discovery.zen.fd.ping_retries: 20
 {% endif %}
 
-{% if elasticsearch_version == '5.6.16' %}
+{% if elasticsearch_version is version('6.0.0', '>=') %}
```

Comment:
Why does this no longer apply to 5.6.16?

Reply:
Ahh! This was a typo. Thanks for catching it. Fixed in https://github.com/dimagi/commcare-cloud/pull/6421/commits/086eba47d387e7a14cd0b99195814e54cd8a3f95